### DISCUSSION TITLE
Pulling Scores/Stats for a Date Range

### DISCUSSION THREAD

**Discussion by ivthoughts** [mattdodge/yahoofantasy#67] [Q&A]
Hey Matt or anyone who has used this library,

For Roto style leagues, is there a way to pull scoring data for a particular date or a date range? Since there are no week objects for a Roto league, I am looking to pull a weeks worth of scoring for Roto teams.

Thanks,

**Comment by mattdodge** [comment]
Interesting question, not immediately aware of that and the [Yahoo docs](https://developer.yahoo.com/fantasysports/guide/) don't have anything obvious either. You could try fiddling with the `/team/<id>/stats` endpoint, in particular some filters or parameters after that, to see if anything works there though

**Reply by ivthoughts** [reply]
Thanks for the quick reply. I will try to play around with it to see if I can figure something out.

**Reply by ivthoughts** [reply]
So I spent some time looking into this and figured out the URL necessary for the call from the docs as you suggested, but using your library, I am not sure if is the right one to use for what I am trying to do. I think due to it being a roto league.

When I try to access the stats of a teams I get the following error:
**'Team' object has no attribute 'stats'**

Additionally, looking at the variables available (vars(team)) for the team object, stats is not an option.
